pkg/controller/worker/machines_test.go

```diff
@@ -358,8 +358,8 @@ var _ = Describe("Machines", func() {
 					},
 				}
 
-				workerPoolHash1, _ = worker.WorkerPoolHash(w.Spec.Pools[0], cluster)
-				workerPoolHash2, _ = worker.WorkerPoolHash(w.Spec.Pools[1], cluster)
+				workerPoolHash1, _ = worker.WorkerPoolHash(w.Spec.Pools[0], cluster, []string{}, []string{})
```

Comment:
any preference of `worker.WorkerPoolHash(w.Spec.Pools[0], cluster, []string{}, []string{})` over `worker.WorkerPoolHash(w.Spec.Pools[0], cluster, nil, nil)`?

Reply:
No, thanks. I adjusted it :)